### PR TITLE
fix Arabic mistakes and add Arabic currency

### PR DESCRIPTION
**Pull request by itsmohmans**
### What does this change
One change to [providers/persons/ar_AA/__init__.py](https://github.com/joke2k/faker/blob/594d34414b/faker/providers/person/ar_AA/__init__.py)
I also added a new currency for Arabic ar_AA

### What was wrong
There was a wrong name in [providers/persons/ar_AA/__init__.py](https://github.com/joke2k/faker/blob/594d34414b/faker/providers/person/ar_AA/__init__.py)

### Note
This is my very first contribution to a project.

### CODE REVIEW

**Comment by fcurella** [comment]
Thank you! Could add some tests for the new currency provider?

**Comment by itsmohmans** [comment]
Can you guide me more on where to add the tests? As I didn't really dig into this project I just localized it. Thanks.

**Comment by fcurella** [comment]
You can copy and modify any of the existing tests for the other currency providers, eg the [Russian](https://github.com/joke2k/faker/blob/master/tests/providers/test_currency.py#L71-L90) one.

**Comment by itsmohmans** [comment]
Done.

File: tests/providers/test_currency.py [hunk 1]
```diff
@@ -68,6 +68,28 @@ def test_cryptocurrency_name(self, faker, num_samples):
...
+    def setup_class(cls):
+        from faker.providers.currency.ar_AA import Provider as ArCurrencyProvider
+        cls.provider = ArCurrencyProvider
+        cls.currencies = cls.provider.currencies
+        cls.currency_codes, cls.currency_names = tuple(zip(*cls.currencies))
+
+    def test_currency(self, faker, num_samples):
+        for _ in range(num_samples):
+            cur = faker.currency()
+            assert isinstance(cur, tuple) and cur in self.currencies
```
Comment (fcurella) [inline tests/providers/test_currency.py:85]: can you split this assertion?

```suggestion
            assert isinstance(cur, tuple)
            assert cur in self.currencies
```

File: tests/providers/test_currency.py [hunk 2]
```diff
@@ -68,6 +68,28 @@ def test_cryptocurrency_name(self, faker, num_samples):
...
+
+    def test_currency_name(self, faker, num_samples):
+        for _ in range(num_samples):
+            name = faker.currency_name()
+            assert isinstance(name, str) and name in self.currency_names
```
Comment (fcurella) [inline tests/providers/test_currency.py:90]: Same as above:

```suggestion
            assert isinstance(name, str)
            assert name in self.currency_names
```

**Review by fcurella — CHANGES_REQUESTED** [review]

**Pull request closed without merging**